### PR TITLE
[YKN-450] : When manually dispatched, the sonar summary link is incorrect

### PR DESCRIPTION
**Pull request by AdityaHitachi**
Fixed the incorrect Sonar link for the GitHub summary and the Slack notification for the manually dispatched event.

### CODE REVIEW

File: action.yaml [hunk 1]
```diff
@@ -222,7 +222,7 @@ runs:
         if [ "${{ env.SONAR_HOST_URL }}" != "" ] && [ "${{ env.SONAR_PROJECT_KEY }}" != "" ]  && [ "${{ steps.all_icons.outputs.sonar_scan }}" != "x" ]; then
             if [ "${{ github.event_name }}" == "pull_request" ]; then
               sonar_result+="pullRequest=${{ github.event.pull_request.number }}"
-            elif [ "${{ github.event_name }}" == "push" ]; then
+            elif [ "${{ github.event_name }}" != "pull_request" ]; then
```
Comment (cardosov) [inline action.yaml:225]: Why not just `else`?

File: action.yaml [hunk 2]
```diff
@@ -293,7 +293,7 @@ runs:
         if [ "${{ env.SONAR_HOST_URL }}" != "" ] && [ "${{ env.SONAR_PROJECT_KEY }}" != "" ]  && [ "${{ steps.all_icons.outputs.sonar_scan }}" != "x" ]; then
           if [ "${{ github.event_name }}" == "pull_request" ]; then
           sonar_result="<${{ env.SONAR_HOST_URL }}/dashboard?id=${{ env.SONAR_PROJECT_KEY }}&pullRequest=${{ github.event.pull_request.number }}|SonarQube>"
-          elif [ "${{ github.event_name }}" == "push" ]; then
+          elif [ "${{ github.event_name }}" != "pull_request" ]; then
```
Comment (cardosov) [inline action.yaml:296]: Same

**Review by cardosov — CHANGES_REQUESTED** [review]

File: action.yaml (hunk 1 above)
Comment (AdityaHitachi) [reply action.yaml:225]: Yes, we can use else also instead elif.

File: action.yaml [hunk 3]
```diff
@@ -293,7 +293,7 @@ runs:
         if [ "${{ env.SONAR_HOST_URL }}" != "" ] && [ "${{ env.SONAR_PROJECT_KEY }}" != "" ]  && [ "${{ steps.all_icons.outputs.sonar_scan }}" != "x" ]; then
           if [ "${{ github.event_name }}" == "pull_request" ]; then
           sonar_result="<${{ env.SONAR_HOST_URL }}/dashboard?id=${{ env.SONAR_PROJECT_KEY }}&pullRequest=${{ github.event.pull_request.number }}|SonarQube>"
```
Comment (cardosov) [inline action.yaml:295]: Just being nitpicking but, please indent the code probably. `sonar_result` should not be at the same level as the the `if` and also in the `else` block.

**Review by cardosov — CHANGES_REQUESTED** [review]

File: action.yaml (hunk 3 above)
Comment (AdityaHitachi) [reply action.yaml]: Indented the code properly.

**Review by cardosov — APPROVED** [review]

**Review by mayur-hitachivantara — APPROVED** [review]

**Review by vganpisetty — APPROVED** [review]

**Pull request merged**